src/interface.jl

```diff
@@ -56,7 +56,14 @@ Broadcast.broadcastable(b::Transform) = Ref(b)
 Transform `x` using `b`, treating `x` as a single input.
 """
 transform(f::F, x) where {F<:Function} = f(x)
-transform(t::Transform, x) = first(with_logabsdet_jacobian(t, x))
+function transform(t::Transform, x)
+    res = with_logabsdet_jacobian(t, x)
+    if res isa ChangesOfVariables.NoLogAbsDetJacobian
+        error("`transform` not implemented for $(typeof(b)); implement `transform` and/or `with_logabsdet_jacobian`.")
```

Comment:
Wouldn't it be more correct to say that logabsdet... is not implemented?At least that's what we check here and why the fallback for transform fails.